docs/tutorials/reading-and-rasterizing-osm-data.rst

```diff
@@ -1,231 +0,0 @@
-
-When we rasterize the ``Feature``\s, we'd like for schools to have
-a different value than all of the other ``Feature``\s.  However, we are unsure
-if any schools were contained within the original data, and we'd like to see
-if any are. One method we could use to determine if there are schools is to
-look at the tags of the ``Way`` ``Feature``\s.  The following code will show
-all of the unique tags for all of the ``Way``\s in the collection.
-
-.. code:: python3
-
-   features.get_way_tags()
-
-Which has the following output:
-
-.. code::
-
-  {'NHD:ComID': '25964938',
-   'NHD:Elevation': '0.00000000000',
-   'NHD:FCode': '46006',
-   'NHD:FDate': '2001/08/16',
-   'NHD:FTYPE': 'LakePond',
-   'NHD:FType': 'StreamRiver',
-   'NHD:Permanent_': '25964312',
-   'NHD:RESOLUTION': 'High',
-   'NHD:ReachCode': '2040203001298',
-   'NHD:Resolution': 'High',
-   'NHD:way_id': '25964938',
-   'NHS': 'yes',
-   'addr:city': 'Boyertown',
-   'addr:housenumber': '709',
-   'addr:postcode': '19512',
-   'addr:street': 'East Philadelphia Avenue',
-   'amenity': 'school',
-   'area': 'yes',
-   'attribution': 'NHD',
-   'bridge': 'yes',
-   'building': 'house',
-   'cuisine': 'pizza',
-   'delivery': 'no',
-   'destination': 'Boyertown;Gilbertsville',
-   'destination:ref': 'PA 73',
-   'dispensing': 'yes',
-   'drive_through': 'yes',
-   'electrified': 'no',
-   'expressway': 'yes',
-   'gauge': '1435',
-   'gnis:feature_id': '2487880',
-   'healthcare': 'optometrist',
-   'hgv': 'yes',
-   'highway': 'service',
-   'landuse': 'cemetery',
-   'lanes': '2',
-   'layer': '1',
-   'leisure': 'stadium',
-   'name': 'East 2nd Street',
-   'name_1': 'Big Road',
-   'name_2': 'Layfield Rd',
-   'natural': 'water',
-   'office': 'accountant',
-   'old_railway_operator': 'Reading',
-   'old_ref:legislative': '197',
-   'oneway': 'yes',
-   'operator': 'Santander',
-   'outdoor_seating': 'no',
-   'railway': 'platform',
-   'ref': 'PA 73',
-   'service': 'alley',
-   'shop': 'beauty',
-   'source': 'Yahoo',
-   'source:hgv': 'PennDOT ftp://ftp.dot.state.pa.us/public/Bureaus/BOMO/MC/Publication411.pdf',
-   'sport': 'american_football',
-   'takeaway': 'yes',
-   'tiger:cfcc': 'A41',
-   'tiger:county': 'Berks, PA',
-   'tiger:name_base': '2nd',
-   'tiger:name_base_1': 'State Route 73',
-   'tiger:name_base_2': 'State Route 73',
-   'tiger:name_base_3': 'Layfield',
-   'tiger:name_direction_prefix': 'E',
-   'tiger:name_type': 'St',
-   'tiger:name_type_1': 'Rd',
-   'tiger:name_type_3': 'Rd',
-   'tiger:reviewed': 'no',
-   'tiger:source': 'tiger_import_dch_v0.6_20070829',
-   'tiger:tlid': '80457400:80457404',
-   'tiger:upload_uuid': 'bulk_upload.pl-6517a127-4250-40d5-88da-8acebe246150',
-   'tiger:zip_left': '19512',
-   'tiger:zip_left_1': '19512',
-   'tiger:zip_right': '19512',
-   'usage': 'freight',
-   'waterway': 'stream'}
-
-
-So it appears that there are schools in this dataset, and that we can continue
-on.
-
-
-Assigning Values to Geometries
```

Comment:
Looks like rasterization docs that we can keep if we  use non OSM data as example.